disk: Add --share-rw option for hotplugging disks

amshinde opened the pull request:
With qemu 2.10, a write lock was added for qcow images that
prevents the same image to be passed more than once.
This can be over-ridden using the --share-rw option which is
desired for raw images.

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

amshinde commented:
cc @mcastelino

markdryan commented on qemu/qmp.go:
```diff
@@ -670,7 +670,7 @@ func (q *QMP) ExecuteBlockdevAdd(ctx context.Context, device, blockdevID string)
 // to a previous call to ExecuteBlockdevAdd.  devID is the id of the device to
 // add.  Both strings must be valid QMP identifiers.  driver is the name of the
 // driver,e.g., virtio-blk-pci, and bus is the name of the bus.  bus is optional.
```
The documentation doesn't mention the new parameter.

amshinde replied:
@markdryan I have addresses this. PTAL.

amshinde commented:
@sboeuf @mcastelino I was going to add details for the Kata issue in a corresponding PR in Kata that passes the shared parameter that I introduced in this PR. Nonetheless, I have added some details here as well. PTAL.

sboeuf commented:
Thanks @amshinde

amshinde commented:
@markdryan Can you take another look at this?
